### DISCUSSION TITLE
Hosting hackathon on Github event

### DISCUSSION THREAD

**Discussion by Sudhanva17** [community/community#50320] [Actions]
### Select Topic Area

Show & Tell

### Body

I wanted to know the steps I should follow to host our University's Hackathon on github events

**Comment by Mohamed-SayedAlAhl** [comment]
To host a hackathon on GitHub Events, you can follow these steps:

1. Create a new repository: Create a new repository on GitHub that will serve as the central hub for your hackathon. You can use this repository to share resources, communicate with participants, and manage submissions.

2. Set up the GitHub Event: Go to the GitHub Events page and create a new event. Fill out the event details, including the date, time, and description of the hackathon.

3. Invite participants: Invite participants to the event by sharing the event link and the repository link. You can also use social media, mailing lists, and other channels to invite participants.

4. Define the hackathon rules and guidelines: Define the rules and guidelines for the hackathon, including the theme, submission requirements, judging criteria, and prizes.

5. Provide resources and support: Provide resources and support to help participants during the hackathon, such as documentation, tutorials, and technical support.

6. Manage submissions: Manage submissions by reviewing and evaluating them according to the judging criteria. You can use GitHub issues or pull requests to manage submissions and provide feedback to participants.

7. Announce winners and wrap up: Announce the winners of the hackathon and wrap up the event by thanking participants, sharing results, and providing feedback for future events.

By following these steps, you can host a successful hackathon on GitHub Events and provide a platform for students and developers to showcase their skills and creativity.

**Reply by Sudhanva17** [reply]
On the GitHub events page I did not find an option to create a new event can you help me on that.

**Reply by Mohamed-SayedAlAhl** [reply]
Oh my info maybe a bit old. I found out that GitHub has discontinued their Events feature, and it is no longer possible to create new events on GitHub. However, you can still use GitHub as a platform for hosting your hackathon by creating a new repository and using it to manage your hackathon. You can use GitHub's issue tracker, pull requests, and wiki to manage your event, communicate with participants, and accept submissions. Additionally, you can use external tools like Devpost or ChallengePost to manage submissions and judging. I hope this information helps.
